cloudvolume/chunks.py

```diff
@@ -151,7 +151,7 @@ def decode_npz(string):
 
 def decode_jpeg(bytestring, shape, dtype):
   img = Image.open(io.BytesIO(bytestring))
-  data = np.array(img.getdata(), dtype=dtype)
+  data = np.asarray(img, dtype=dtype).ravel()
```

Comment:
should be `ravel(order='F')`

Reply:
Default order is C: https://numpy.org/doc/stable/reference/generated/numpy.ravel.html